homeassistant/components/tradfri/light.py

```diff
@@ -52,27 +52,18 @@ def __init__(self, device, api, gateway_id):
         """Initialize a Group."""
         super().__init__(device, api, gateway_id)
 
-        self._unique_id = f"group-{gateway_id}-{device.id}"
-
+        self._attr_unique_id = f"group-{gateway_id}-{device.id}"
+        self._attr_should_poll = True
+        self._attr_supported_features = SUPPORTED_GROUP_FEATURES
```

Comment:
These are static, move them to class attributes on the `TradfriGroup` class

Reply:
Done.